pkg/server/framework/request.go

```diff
@@ -95,3 +96,20 @@ func Decode(r *http.Request, val interface{}) error {
 
 	return nil
 }
+
+// Post does a post request with data to provided endpoint
+func Post(endpoint string, data interface{}) (*http.Response, error) {
+	// convert response payload to json
+	jsonData, err := json.Marshal(data)
+	if err != nil {
```

Comment:
consider a meaningful response to the client if they haven't provided a json request body

Reply:
Yup I was thinking about doing this, but wanted the real response object from the dwn before I expanded upon this all the way. 

I will put in a response object on our side and guestimate what the dwn response would be